### PR TITLE
[EZ] Lambda Tagging

### PR DESCRIPTION
**Pull request by amarjandu**
Adds resource tagging on AWS Lambda Functions

### CODE REVIEW

File: chalice/build_deploy_config.sh [hunk 1]
```diff
@@ -43,7 +49,11 @@ else
 fi
 
 export DEPLOY_ORIGIN="$(whoami)-$(hostname)-$(git describe --tags --always)-$(date -u +'%Y-%m-%d-%H-%M-%S').deploy"
-cat "$config_json" | jq .stages.$stage.tags.DSS_DEPLOY_ORIGIN=env.DEPLOY_ORIGIN | sponge "$config_json"
+cat "$config_json" | jq ".stages.$stage.tags.DSS_DEPLOY_ORIGIN=env.DEPLOY_ORIGIN | \
+	.stages.$stage.tags.Name=env.dss_infra_tag_name |  .stages.$stage.tags.project=env.dss_infra_tag_project | \
```
Comment (xbrianh) [inline chalice/build_deploy_config.sh]: nit: remove double space

File: daemons/build_deploy_config.sh [hunk 2]
```diff
@@ -31,7 +36,11 @@ fi
 cat "$config_json" | jq ".stages.$stage.api_gateway_stage=env.stage" | sponge "$config_json"
 
 export DEPLOY_ORIGIN="$(whoami)-$(hostname)-$(git describe --tags --always)-$(date -u +'%Y-%m-%d-%H-%M-%S').deploy"
-cat "$config_json" | jq .stages.$stage.tags.DSS_DEPLOY_ORIGIN=env.DEPLOY_ORIGIN | sponge "$config_json"
+cat "$config_json" | jq ".stages.$stage.tags.DSS_DEPLOY_ORIGIN=env.DEPLOY_ORIGIN | \
+	.stages.$stage.tags.Name=env.dss_infra_tag_name |  .stages.$stage.tags.project=env.dss_infra_tag_project | \
```
Comment (xbrianh) [inline daemons/build_deploy_config.sh]: nit: remove double space

File: chalice/build_deploy_config.sh [hunk 3]
```diff
@@ -8,6 +8,12 @@ if [[ -z $DSS_DEPLOYMENT_STAGE ]]; then
 fi
 
 export stage=$DSS_DEPLOYMENT_STAGE
+export dss_infra_tag_project=${DSS_INFRA_TAG_PROJECT}
+export dss_infra_tag_service=${DSS_INFRA_TAG_SERVICE}
+export dss_infra_tag_stage=${DSS_DEPLOYMENT_STAGE}
+export dss_infra_tag_owner=${DSS_INFRA_TAG_OWNER}
+export dss_infra_tag_name=${DSS_INFRA_TAG_PROJECT}-${DSS_DEPLOYMENT_STAGE}-${DSS_INFRA_TAG_SERVICE}
```
Comment (xbrianh) [inline chalice/build_deploy_config.sh]: What's the purpose of renaming these variables?

Comment (amarjandu) [reply chalice/build_deploy_config.sh]: its to evaluate the env variable  in JQ without having to mess about with quote escaping, i realize that i had to perform it this way when i first started, i may be able to change this.

File: daemons/build_deploy_config.sh (hunk 2 above)
Comment (amarjandu) [reply daemons/build_deploy_config.sh]: will remove

File: chalice/build_deploy_config.sh (hunk 1 above)
Comment (amarjandu) [reply chalice/build_deploy_config.sh]: will remove

**Pull request merged**